### ISSUE TITLE
virus defend by windows, it`s a black windows 

### ISSUE THREAD

**Issue by banrieen** [SerGreen/Appacker#15]
virus defend by windows defend. and even I disable the defend, finish the pack, but it is only a black windows when click the output exe 

and no logs can found.
![image](https://user-images.githubusercontent.com/13807893/149121665-634dfd35-b227-492b-ac26-19697eb92774.png)

**Comment by SerGreen** [comment]
Indeed, for me too Windows Defender detects it as a virus again. It's a false positive, i reported it as such to Microsoft several times, and they fixed it several times, and here it's doing it again. It even tries to block the resulting output file.  

On the screenshot it says that packaging wasn't completed successfully, so my guess is that Windows Defender put it in the quarantine before it could finish so the output.exe is corrupted now. I suggest you to re-download the Appacker.exe and try packaging again.  

And i should add logging i suppose, as well as report the false positive to MS.

**Comment by SerGreen** [comment]
**Update:**  
Got a response from MS, false detection has been removed from Windows Defender, so it should no longer be detected as a virus. You might want to update Defender's malware definitions manually for the fix to apply immediately. Quote from the MS reply:  

>Please follow the steps below to clear cached detections and obtain the latest malware definitions.
>  
>1. Open command prompt as administrator and change directory to c:\Program Files\Windows Defender
>2. Run “MpCmdRun.exe -removedefinitions -dynamicsignatures”
>3. Run "MpCmdRun.exe -SignatureUpdate"
